src/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Document</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+<div class="red"></div>
```

Comment:
don't forget about whitespaces
```suggestion
  <div class="red"></div>
```